README.md

```diff
@@ -14,6 +14,30 @@ This will stand up a cluster logging stack named 'example'.
 
 ## Hacking
 
+### Building a Universal Base Image (UBI) based image
+
+You must first `oc login api.ci.openshift.org`.  You'll need these credentials in order
+to pull images from the UBI registry.
+
+The image build process for UBI based images uses a private yum repo.
+In order to use the private yum repo, you will need access to
+https://github.com/openshift/release/blob/master/cluster/ci/config/prow/openshift/rpm-mirrors/ocp-4.0-default.repo
+and
+https://github.com/openshift/shared-secrets/blob/master/mirror/ops-mirror.pem
+Note that the latter is private and requires special permission to access.
+
+The best approach is to clone these repos under `$GOPATH/src/github.com/openshift`
+which the build scripts will pick up automatically.  If you do not, the build script
+will attempt to clone them to a temporary directory.
+
+### Running a full integration test with a 4.x cluster
+
+If you have a local clone of [origin-aggregated-logging](https://github.com/openshift/origin-aggregated-logging)
+under `$GOPATH/github.com/openshift/origin-aggregated-logging` you can use its `hack/get-cluster-run-tests.sh`
+to build the logging, elasticsearch-operator, and cluster-logging-operator images; deploy a 4.x cluster;
+push the images to the cluster; deploy logging; and launch the logging CI tests.
```

Comment:
How I can specify the repo and branch for origin-aggregated-logging?
Could there be a pair of options like this?  
LOGGING_GITHUB_REPO=nhosoi LOGGING_FORK_BRANCH=test_branch
Or $GOPATH/github.com/openshift/origin-aggregated-logging is used as it is?  (no git push needed?)

Reply:
`$GOPATH/github.com/openshift/origin-aggregated-logging` is used as is - so it will build/run whatever you have checked out and working on.  No git push needed.  It will build the images locally from your working directory and push directly to the remote repo.